packages/terra-aggregator/examples/tmp/NewModalManager.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
This file will be moved into the terra-modal-manager package and replace the existing ModalManager.